dist/fabric.js

```diff
@@ -17637,6 +17741,22 @@ fabric.util.object.extend(fabric.Object.prototype, /** @lends fabric.Object.prot
       return this;
     },
 
+    /**
+     * Draws something special to highlight an indicated object.
+     * Requires public properties: width, height
+     * Requires public options: cornerSize, padding
+     * @param {CanvasRenderingContext2D} ctx Context to draw on
+     * @param {Object} styleOverride object to override the object style
+     * @return {fabric.Object} thisArg
+     * @chainable
+     */
+    drawIndication: function(ctx, styleOverride) {
+      styleOverride = styleOverride || {};
+      this.drawIndicationBorders(ctx, styleOverride);
+      return this;
+    },
+
+
```

Comment:
this also.

Reply:
Same as above (funny that the documentation is more lines than the actual code!)